docs/docs/widgets/toggle-switch.md

```diff
@@ -79,8 +79,8 @@ Under the <b>General</b> accordion, you can set the value in the string format.
 
 |  <div style={{ width:"100px"}}> Style </div> |  <div style={{ width:"100px"}}> Description </div> |  <div style={{ width:"100px"}}> Default Value </div>|
 |:----- |:---------  |:------------- |
-| Text color | Change the color of the text in the widget by providig the `Hex color code` or choosing a color from the picker. |  |
-| Toggle switch color | Change the color of the toggle switch in the widget by providig the `Hex color code` or choosing a color from the picker. |  |
+| Text color | Change the color of the text in the widget by providing the `Hex color code` or choosing a color from the picker. |  |
```

Comment:
Please check if the formatting is correct here, I see double bars `| |` in the table.